src/background/services/network/NetworkService.ts

```diff
@@ -466,7 +466,9 @@ export class NetworkService implements OnLock, OnStorageReady {
   ): Promise<NetworkWithCaipId | undefined> {
     const chainId =
       typeof scopeOrChainId === 'string'
-        ? caipToChainId(scopeOrChainId)
+        ? scopeOrChainId.startsWith('0x')
```

Comment:
Please add a test for the hex number case

Reply:
Added tests for entire `.getNetwork()` in d2632f72b